packages/map/src/GMap.ts

```diff
@@ -822,8 +825,11 @@ export interface GMap {
     googleMapStyles_exists(): boolean;
     useComputedHeading(): boolean;
     useComputedHeading(_: boolean): this;
+    hideStreetViewMarker(): boolean;
+    hideStreetViewMarker(_: boolean): this;
 }
 
+GMap.prototype.publish("hideStreetViewMarker", true, "boolean", "If true, streetView marker will be hidden");
```

Comment:
Rename to "showStreetViewMarker" and make the default match the behaviour prior to this PR.